tests/common.py

```diff
@@ -6,7 +6,7 @@
 import torch
 
 
-class TestDataset(torch.utils.data.Dataset):
+class BoringDataset(torch.utils.data.Dataset):
```

Comment:
Nice :D

Reply:
`pytest` attempts to parse it as a test everything that starts with `Test`. It understands that it is not a test but still prints a message about it.